1-js/05-data-types/02-number/article.md

```diff
@@ -409,10 +410,10 @@ There are more functions and constants in `Math` object, including trigonometry,
 
 ## Summary
 
-To write big numbers:
+To write numbers with many zeroes:
 
-- Append `"e"` with the zeroes count to the number. Like: `123e6` is `123` with 6 zeroes.
-- A negative number after `"e"` causes the number to be divided by 1 with given zeroes. That's for one-millionth or such.
+- Append `"e"` with the zeroes count to the number. Like: `123e6` is the same as `123` with 6 zeroes `123000000`.
```

Comment:
I thought it was OK the way it was, but if you're going to do it this way, it would be better with a colon after "zeroes":

"the same as `123` with 6 zeroes: `12300000`.